### PR TITLE
Removing references to pre v1beta3 apis in code

### PR DESCRIPTION
**Pull request by nikhiljindal**

### CODE REVIEW

**Comment by nikhiljindal** [comment]
@caesarxuchao @bgrant0607 @krousey

File: pkg/api/meta/restmapper.go [hunk 1]
```diff
@@ -66,8 +66,7 @@ type typeMeta struct {
 // can be mapped with the provided MetadataAccessor and Codec interfaces.
 //
 // The resource name of a Kind is defined as the lowercase,
-// English-plural version of the Kind string in v1beta3 and onwards,
-// and as the camelCase version of the name in v1beta1 and v1beta2.
+// English-plural version of the Kind string,
```
Comment (caesarxuchao) [inline pkg/api/meta/restmapper.go]: s/,/.

Comment (nikhiljindal) [inline pkg/api/meta/restmapper.go]: Good catch, Fixed

File: pkg/client/nodes.go [hunk 2]
```diff
@@ -49,12 +49,8 @@ func newNodes(c *Client) *nodes {
...
 
-// resourceName returns node's URL resource name based on resource version.
-// Uses "minions" as the URL resource name for v1beta1 and v1beta2.
+// resourceName returns node's URL resource name.
 func (c *nodes) resourceName() string {
-	if api.PreV1Beta3(c.r.APIVersion()) {
-		return "minions"
-	}
 	return "nodes"
 }
```
Comment (caesarxuchao) [inline pkg/client/nodes.go]: Can we replace all the calls to this function with "nodes"? It's only used in this file.

**Comment by caesarxuchao** [comment]
LTGM except a nit.

**Comment by krousey** [comment]
LGTM

File: pkg/client/nodes.go (hunk 2 above)
Comment (nikhiljindal) [inline pkg/client/nodes.go]: Letting it be. I can see us changing that again :)

Comment (caesarxuchao) [inline pkg/client/nodes.go]: Ok. LGTM. Thanks. @krousey since you have also given a LGTM, I'll set the LGTM label.

**Comment by nikhiljindal** [comment]
Assigning to @bgrant0607 for ok-to-merge.

**Comment by bgrant0607** [comment]
I think this can wait. We're trying to retain the ability to easily merge critical fixes into 1.0 for now.

**Comment by nikhiljindal** [comment]
cc @vishh for merging

**Comment by vishh** [comment]
@k8s-bot: test this please

**Pull request merged**